README.md

```diff
@@ -30,6 +30,8 @@ By default, the limitation is common to all routes annotated `@RateLimit()`.
 For example, if you keep the default configuration and you configure the `@RateLimit()` annotation in 2 routes. Limit will shared between this 2 routes, if user consume all authorized calls on the first route, the second route couldn't be called.
 If you swicth `limit_by_route` to true, users will be allowed to reach the limit on each route annotated.
 
+`@GraphQLRateLimit()`annotation allows you to rate limit by graphQL query or mutation.
+
 If you switch `display_headers` to true, 3 headers will be added `x-rate-limit`, `x-rate-limit-hits`, `x-rate-limit-untils` to your responses. This can be usefull to debug your limitations.
```

Comment:
```suggestion
If you switch `display_headers` to true, 3 headers will be added `x-rate-limit`, `x-rate-limit-hits`, `x-rate-limit-untils` to your responses. This can be usefull to debug your limitations.
It also let the API's callers adjust their calls based on the limitation / remaining calls returned by the application.
```

Reply:
This is not really related to this PR and also I am not really confident with this. It doesn't seem a good idea to expose this kind of information because someone could bypass the rate limitation, that's why it is more a 'dev' feature.

Reply:
ℹ️ I just added this to talk about headers interpreted by the api callers. I don't see how someone could bypass the rate limitation in place 🤔 . It will just "display" the headers in header response.
Just an idea. It can be removed or placed in another pull request 😄